src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -57,12 +62,128 @@ trait WorkspaceComponent {
     def workspace = foreignKey("FK_WS_ACCESS_WORKSPACE", workspaceId, workspaceQuery)(_.id)
     def group = foreignKey("FK_WS_ACCESS_GROUP", groupName, rawlsGroupQuery)(_.groupName)
 
-    def x = primaryKey("PK_WORKSPACE_ACCESS", (workspaceId, accessLevel))
+    def accessPrimaryKey = primaryKey("PK_WORKSPACE_ACCESS", (workspaceId, accessLevel))
 
     def * = (workspaceId, groupName, accessLevel) <> (WorkspaceAccessRecord.tupled, WorkspaceAccessRecord.unapply)
   }
 
-  protected val workspaceQuery = TableQuery[WorkspaceTable]
   protected val workspaceAttributeQuery = TableQuery[WorkspaceAttributeTable]
   protected val workspaceAccessQuery = TableQuery[WorkspaceAccessTable]
+
+  object workspaceQuery extends TableQuery(new WorkspaceTable(_)) {
+    def workspaceAccessGroups(workspaceId: UUID) = for {
+      workspaceRec <- workspaceQuery if workspaceRec.id === workspaceId
```

Comment:
Why is this query needed?  The workspaceAccessQuery call alone looks sufficient.